binr/r2pm/r2pm

```diff
@@ -583,23 +568,25 @@ case "$1" in
 	if [ -n "$2" ]; then
 		RC=0
 		while [ -n "$2" ]; do
-			echo "Cleaning $2..."
+			echo "Cleaning $2..." >&2
```

Comment:
imho this is a status message, not an error/warning and those should be printed to stduot, not stderr

Reply:
ok 6599f70acb4bc888b143af7e888f729690ece654